doc/usage.md

```diff
@@ -32,7 +32,7 @@ psql
 In the PostgreSQL command-line shell, type:
 
 ```sql
-CREATE USER compass WITH SUPERUSER LOGIN PASSWORD 'mypassword';
+CREATE USER compass WITH SUPERUSER LOGIN PASSWORD <mypassword>;
```

Comment:
Are you sure this will work? The documentation specifies that passwords are strings, and as such, shall be in `'`s: https://www.postgresql.org/docs/current/sql-createrole.html

Reply:
Well I changed that one for consistency, because everywhere else we use <string>, but I suppose we could use '<string>' here?

Reply:
Everywhere else, we specify *variables*, and those variables are paths which rarely (on UNIX) contain spaces and unambiguous with regards to capitalisation.
However, I am not sure if postgres even **supports** specifying this thing *wihout* the quotes, i.e. will its parser know we are giving a string option, and not something else.

PostgreSQL's string vs. entityname handling is really whack, *especially* in the `psql` shell.